forc-pkg/src/pkg.rs

```diff
@@ -1215,7 +1215,7 @@ where
     if tags {
         fetch_opts.download_tags(git2::AutotagOption::All);
     }
-    repo.remote_anonymous(source.repo.as_str())?
+    repo.remote("origin", source.repo.as_str())?
```

Comment:
Now that this is no longer an anonymous remote, perhaps we should make this a constant and re-use it wherever "origin" appears?